tests/behavior/transforms/remap.toml

```diff
@@ -1225,3 +1225,28 @@
     [[tests.outputs.conditions]]
     "a.equals" = "fbaro"
     "b.equals" = "fbarbar"
+
+[transforms.remap_metrics]
+  inputs = []
+  type = "remap"
+  source = """
+      .tags.host = "ook"
+      .tags.name = .name
+      .tags.namespace = .namespace
```

Comment:
This doesn't appear to be testing everything we want it to test, right?

That is, it's only testing inserting `tags` here, whereas we should also test`.namespace = xxx`, `.name = xxx` etc.

(I assume nesting `namespace` and `name` under `tags` was the oversight here?)

Reply:
It seems that for metric events, the default contition only checks tags - it doesn't work with any other fields of the metric.

I could, of course use the Remap condition which can check all the relevant fields for the event - but it feels wrong to use the thing that you are testing to test the thing.